data/korath ships.txt

```diff
@@ -185,6 +191,7 @@ ship "Korath World-Ship"
 		"Large Heat Shunt" 2
 		"Fuel Processor"
 		"Korath Repeater Rifle" 150
+		"Outfits Expansion"
```

Comment:
The stock WS doesn't need this because it isn't receiving anything else.

Reply:
Removed.